addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -383,6 +383,7 @@ var ListRenderer = BasicRenderer.extend({
     _renderBodyCell: function (record, node, colIndex, options) {
         var tdClassName = 'o_data_cell';
         if (node.tag === 'button') {
+            delete node.attrs.width;
```

Comment:
why do you need this?

Reply:
The width set for td cell is getting applied on the button so that's why I have removed the width.